e2e/cases/dts/index.test.ts

```diff
@@ -1,20 +1,76 @@
 import { join } from 'node:path';
 import { buildAndGetResults } from '@e2e/helper';
-import { expect, test } from 'vitest';
+import { describe, expect, test } from 'vitest';
 
-test('dts when bundle: false', async () => {
-  const fixturePath = join(__dirname, 'bundle-false');
-  const { files, contents } = await buildAndGetResults(fixturePath, 'dts');
+describe('dts when bundle: false', () => {
+  test('basic - bundleless dts', async () => {
+    const fixturePath = join(__dirname, 'bundle-false');
+    const { files, contents } = await buildAndGetResults(
+      fixturePath,
+      'rslib.config.ts',
+      'dts',
+    );
 
-  expect(files.esm?.length).toBe(4);
-  expect(files.esm?.[0]!.endsWith('.d.ts')).toEqual(true);
-  expect(contents.esm).toMatchSnapshot();
+    expect(files.esm?.length).toBe(4);
+    expect(files.esm?.[0]!.endsWith('.d.ts')).toEqual(true);
+    expect(contents.esm).toMatchSnapshot();
+  });
+
+  test('dts false - bundleless dts', async () => {
+    const fixturePath = join(__dirname, 'bundle-false');
+    const { files } = await buildAndGetResults(
+      fixturePath,
+      'rslib.false.config.ts',
+      'dts',
+    );
+
+    expect(files.esm).toBe(undefined);
+  });
+
+  test('distPath - bundleless dts', async () => {
+    const fixturePath = join(__dirname, 'bundle-false');
+    const { files } = await buildAndGetResults(
+      fixturePath,
+      'rslib.distpath.config.ts',
+      'dts',
+    );
+    expect(files.esm?.length).toBe(4);
+    expect(files.esm?.[0]!.startsWith('./dist/custom')).toEqual(true);
```

Comment:
Is the order in files stable?

Reply:
The order doesn't matter, we only need to check any file whether it located in correct distpath or not.